tailwind.config.js

```diff
@@ -11,12 +11,15 @@ module.exports = {
         'custom-blue': {
           blue: '#2572B4',
           light: '#1492B4',
-          dark: '#096684',
+          dark: '#4C5E7F',
         },
         "gray-light": "#EFEFEF",
         "gray-md": "#CFCFCF",
         'dk-blue': '#26374a',
-        'hr-red-bar': '#AE3B43'
+        'hr-red-bar': '#AE3B43',
+        //Search bar colors
+        "search-text-gray": "#5c5c5c",
```

Comment:
these colours are not specific to the search bar so not sure it should be called search-button-[colour]